Sources/Fullscreen/Stories/StoriesView.swift

```diff
@@ -73,8 +88,14 @@ public class StoriesView: UIView {
 
     // MARK: - Public methods
 
-    public func configure(with stories: [StoryViewModel]) {
+    public func configure(
+        with stories: [StoryViewModel],
+        startIndex: Int,
+        feedbackViewModel: StoryFeedbackViewModel? = nil
+    ) {
         self.stories = stories
+        self.startIndex = startIndex
```

Comment:
I guess `configure(with: ...)` will only be called once and before the first `layoutSubviews`, but since it might be used more times (?) and you use the `startIndex` in `layoutSubviews` maybe a call to `setNeedsLayout` is good to have here? 🤔

Reply:
yes, at the moment it's used only once. Good point, I added it. Now it should work well even if our needs change in the future 😄 👍